AUTHORS.md

```diff
@@ -1,10 +1,10 @@
 # Core Developer #
 
 ----------
-- Sepand Haghighi - Sharif University Of Technology/Moduland Co - [@sepandhaghighi](http://github.com/sepandhaghighi)
-- Sadra Sabouri - Sharif University Of Technology - [@sadrasabouri](https://github.com/sadrasabouri)
+- Sepand Haghighi - Sharif University of Technology - [@sepandhaghighi](http://github.com/sepandhaghighi)
+- Sadra Sabouri - Sharif University of Technology - [@sadrasabouri](https://github.com/sadrasabouri)
 
 # Other Contributors #
-- Mohammad Hosein Moslemi - Sharif University Of Technology - [@mhmoslemi2338](https://github.com/mhmoslemi2338)
+- Mohammad Hosein Moslemi - [@mhmoslemi2338](https://github.com/mhmoslemi2338)
```

Comment:
Why did you delete this?
He is from Sharif University, no problem.

Reply:
It's all about the format, nothing else